plugins/community/roadrunner-server-php-grpc/v4.4.0/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1.6
+FROM golang:1.21.1-bullseye AS build
+RUN git clone --depth=1 --branch v4.4.0 https://github.com/roadrunner-server/grpc.git
```

Comment:
Changed this to clone then build since doing a `go install` directly from the tag isn't working this release. See https://github.com/bufbuild/plugins/pull/656#discussion_r1261705182 for history here.

cc @rustatian

Reply:
Hey @pkwarren 👋🏻 
This is strange because v4.3.0 works fine, while v4.3.1 does not...

Reply:
I think it was just missing the `protoc_plugins/v4/protoc-gen-php-grpc/v4.4.0` tag. Not a big deal though - this should work going forward.

Reply:
Since there is a nested go.mod, and the Go module in that nested file is

```
github.com/roadrunner-server/grpc/protoc_plugins/v4
```

You would go install ...

```
go install github.com/roadrunner-server/grpc/protoc_plugins/v4/protoc-gen-php-grpc@latest
```

Not the location of the /v4